### DISCUSSION TITLE
Does Gruntwork have a DocumentDB module?

### DISCUSSION THREAD

**Discussion by iantybo** [gruntwork-io/knowledge-base#323] [Help] [answered]
Does Gruntwork have a DocumentDB module?

**Comment by gruntwork-discussions** [comment]
Please add at least one of the following repositories in your initial post (URL or plain text). Our bot will pick it up and add the corresponding label.

* [r:terraform-aws-service-catalog](https://github.com/gruntwork-io/terraform-aws-service-catalog)
* [r:terraform-aws-vpc](https://github.com/gruntwork-io/terraform-aws-vpc)
* [r:terraform-aws-architecture-catalog](https://github.com/gruntwork-io/terraform-aws-architecture-catalog)
* [r:terraform-aws-asg](https://github.com/gruntwork-io/terraform-aws-asg)
* [r:terraform-aws-beanstalk](https://github.com/gruntwork-io/terraform-aws-beanstalk)
* [r:terraform-aws-cache](https://github.com/gruntwork-io/terraform-aws-cache)
* [r:terraform-aws-ci](https://github.com/gruntwork-io/terraform-aws-ci)
* [r:terraform-aws-ci-pipeline-example](https://github.com/gruntwork-io/terraform-aws-ci-pipeline-example)
* [r:terraform-aws-cis-service-catalog](https://github.com/gruntwork-io/terraform-aws-cis-service-catalog)
* [r:terraform-aws-data-storage](https://github.com/gruntwork-io/terraform-aws-data-storage)
* [r:terraform-aws-ecs](https://github.com/gruntwork-io/terraform-aws-ecs)
* [r:terraform-aws-eks](https://github.com/gruntwork-io/terraform-aws-eks)
* [r:terraform-aws-elk](https://github.com/gruntwork-io/terraform-aws-elk)
* [r:terraform-aws-kafka](https://github.com/gruntwork-io/terraform-aws-kafka)
* [r:terraform-aws-lambda](https://github.com/gruntwork-io/terraform-aws-lambda)
* [r:terraform-aws-load-balancer](https://github.com/gruntwork-io/terraform-aws-load-balancer)
* [r:terraform-aws-messaging](https://github.com/gruntwork-io/terraform-aws-messaging)
* [r:terraform-aws-mongodb](https://github.com/gruntwork-io/terraform-aws-mongodb)
* [r:terraform-aws-monitoring](https://github.com/gruntwork-io/terraform-aws-monitoring)
* [r:terraform-aws-openvpn](https://github.com/gruntwork-io/terraform-aws-openvpn)
* [r:terraform-aws-sam](https://github.com/gruntwork-io/terraform-aws-sam)
* [r:terraform-aws-security](https://github.com/gruntwork-io/terraform-aws-security)
* [r:terraform-aws-server](https://github.com/gruntwork-io/terraform-aws-server)
* [r:terraform-aws-static-assets](https://github.com/gruntwork-io/terraform-aws-static-assets)
* [r:terraform-aws-zookeeper](https://github.com/gruntwork-io/terraform-aws-zookeeper)
* [r:terragrunt](https://github.com/gruntwork-io/terragrunt)
* [r:terratest](https://github.com/gruntwork-io/terratest)
* [r:repo-copier](https://github.com/gruntwork-io/repo-copier)
* [r:aperture](https://github.com/gruntwork-io/aperture)


[Community Guidelines](https://github.com/gruntwork-io/knowledge-base/discussions/1)
[Getting help](https://gruntwork.io/support/)
[Gruntwork documentation](https://www.gruntwork.io)

**Comment by rhoboat** [comment, marked as answer]
No we don't. Supporting DocumentDB is not planned, but there's an issue for it [here](https://github.com/gruntwork-io/terraform-aws-data-storage/issues/81).
